### PR TITLE
Upload file to Fluence IPFS sidecar

### PR DESCRIPTION
**Pull request by DieMyst**
A feature to upload files to IPFS (first step for uploading and starting services in Fluence network)
`aqua upload --path path/to/file --addr /multiaddr/to/node`

### CODE REVIEW

File: npm/src/compiled/ipfs.ts [hunk 1]
```diff
@@ -0,0 +1,674 @@
+/**
+ *
+ * This file is auto-generated. Do not edit manually: changes may be erased.
+ * Generated by Aqua compiler: https://github.com/fluencelabs/aqua/.
+ * If you find any bugs, please write an issue on GitHub: https://github.com/fluencelabs/aqua/issues
+ * Aqua version: 0.5.1-SNAPSHOT
+ *
+ */
+import { Fluence, FluencePeer } from '@fluencelabs/fluence';
```
Comment (alari) [inline npm/src/compiled/ipfs.ts:9]: Why do you need it?

File: cli/.js/src/main/scala/aqua/builder/Console.scala [hunk 2]
```diff
@@ -29,6 +29,7 @@ class Console(serviceId: String, fnName: String) extends ServiceFunction {
         // if an input function returns a result, our success will be after it is printed
         // otherwise finish after JS SDK will finish sending a request
         OutputPrinter.print(str)
+        Dynamic.literal()
```
[the remark this answers is not in the document]
Comment (DieMyst) [reply cli/.js/src/main/scala/aqua/builder/Console.scala:32]: it returns empty js object, i'll add a comment

[the remark this answers is not in the document]
Comment (alari) [reply cli/.js/src/main/scala/aqua/builder/Console.scala:32]: Will `Map.empty` work?

[the remark this answers is not in the document]
Comment (DieMyst) [reply cli/.js/src/main/scala/aqua/builder/Console.scala]: no

[the remark this answers is not in the document]
Comment (alari) [reply cli/.js/src/main/scala/aqua/builder/Console.scala]: Could you elaborate?

[the remark this answers is not in the document]
Comment (DieMyst) [reply cli/.js/src/main/scala/aqua/builder/Console.scala]: no, it won't work, cause `Map.empty` != `{}` in JS. `{}` in Scala not equal `{}` in JS too

**Comment by alari** [comment]
Related to #379

**Review by alari — APPROVED** [review]

**Pull request merged**